libc/include/llvm-libc-macros/math-function-macros.h

```diff
@@ -9,10 +9,14 @@
 #ifndef LLVM_LIBC_MACROS_MATH_FUNCTION_MACROS_H
 #define LLVM_LIBC_MACROS_MATH_FUNCTION_MACROS_H
 
+#include "math-macros.h"
```

Comment:
Can you also update the dependency in `CMakeLists.txt`?

Reply:
done.